cms/static/sass/partials/_variables.scss

```diff
@@ -278,3 +278,9 @@ $body-line-height: golden-ratio(.875em, 1);
 // carried over from LMS for xmodules
 $action-primary-active-bg: #1AA1DE; // $m-blue
 $very-light-text: $white;
+
+$color-border: #D9D9D9;
+$color-background: #F2F8FB;
```

Comment:
This is not not the standard background color for Studio which is ``$uxpl-grayscale-x-back`` aka ``#f5f5f5``. Can it just use the standard color? We shouldn't be using any colors that are not in the pattern library.

If we really require a new shade of gray, I wouldn't call this variable ``$color-background`` as that is confusing. I'd try to name it for where it is used.